app/scripts/filters/util.js

```diff
@@ -322,6 +322,11 @@ angular.module('openshiftConsole')
       return limitToFilter(input, limit);
     };
   })
+  .filter("getWebhookSecretData", function() {
+    return function(webhookTrigger) {
+      return webhookTrigger.data[_.toLower(webhookTrigger.data.type)];
```

Comment:
Probably better to use _.get to avoid errors if webhookTrigger is not defined

```js
var type = _.get(webhookTrigger, 'data.type');
if (!type) {
  return null;
}
return _.get(webhookTrigger, ['data', _.toLower(type)]);
```